meson.build

```diff
@@ -195,16 +195,9 @@ if get_option('plugin_altos')
 endif
 
 if get_option('plugin_uefi')
-  fwup = dependency('fwup', version : '>= 10')
-  if fwup.version().version_compare('>= 11')
-    conf.set('HAVE_FWUP_GET_ESP_MOUNTPOINT', '1')
-  endif
-  if fwup.version().version_compare('>= 12')
-    conf.set('HAVE_FWUP_VERSION', '1')
-  endif
   efivar = dependency('efivar')
```

Comment:
I think that as part of dropping `fwup` we would have to pick up the correct _minimum_ `efivar` as a dependency now.

Reply:
Do you know what the minimum required efivar is?

Reply:
33

Reply:
Thanks, fixed.